app/views/tweets/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= turbo_frame_tag "modal" do %>
```

Comment:
Maybe you could rename this tag to a more descriptive one like "new-tweet-modal"